.travis.yml

```diff
@@ -4,13 +4,20 @@ node_js:
 dist: trusty
 git:
   depth: false
+script:
+  - npm run build
 jobs:
   include:
-    - stage: "Fundamental-react: Lint"
-      script: npm run lint
-    - stage: "Fundamental-react: Test"
-      name: "Unit tests"
-      script: npm run test:coverage
+  - stage: "Fundamental-react: Test & Lint"
+    script: npm run lint && npm run test:coverage
+  - stage: Deploy
+    if: branch = master AND type != pull_request
```

Comment:
Yes, every pull request to master comes through as master and pull_request, so we don't want those causing deployments.

Reply:
These are pull requests: 
<img width="1208" alt="screen shot 2019-01-09 at 1 53 54 pm" src="https://user-images.githubusercontent.com/29607818/50930941-27bc5200-1416-11e9-97bf-09fa7a873696.png">

The only way to get around this is to _not_ build on pr's, but this would cause problems for contributions coming from forks.

Reply:
Sounds good.